src/analysis/safety_assertion_mode.rs

```diff
@@ -13,6 +14,16 @@ impl Default for SafetyAssertionMode {
     }
 }
 
+impl fmt::Display for SafetyAssertionMode {
```

Comment:
@sdroege Use Display for code snippets is acceptable for you?

Reply:
No, but your decision :)

Reply:
@GuillaumeGomez  can you move this match as function to writer?
IMHO just to "mod.rs" will do.

Reply:
Ok!